src/sample/index.ts

```diff
@@ -21,33 +23,60 @@ export class ExampleStack extends cdk.Stack {
       handler: "hello.handler",
     });
 
+    // set up rest api and log group
+    const restLogGroup = new LogGroup(this, "restLogGroup");
+    new LambdaRestApi(this, "rest-test", {
+      handler: hello,
+      deployOptions: {
+        accessLogDestination: new LogGroupLogDestination(restLogGroup),
+      },
+    });
+
     const hello1 = new lambda.Function(this, "HelloHandler1", {
       runtime: lambda.Runtime.NODEJS_10_X,
       code: lambda.Code.fromAsset("./src/sample/lambda"),
       handler: "hello.handler",
     });
 
+    const restLogGroup1 = new LogGroup(this, "restLogGroup1");
+    new LambdaRestApi(this, "rest-test1", {
+      handler: hello,
+      deployOptions: {
+        accessLogDestination: new LogGroupLogDestination(restLogGroup1),
+      },
+    });
+
     const hello2 = new lambda.Function(this, "HelloHandler2", {
       runtime: lambda.Runtime.PYTHON_3_7,
       code: lambda.Code.fromAsset("./src/sample/lambda"),
       handler: "hello_py.handler",
       tracing: lambda.Tracing.ACTIVE,
     });
 
+    const restLogGroup2 = new LogGroup(this, "restLogGroup2");
+    new LambdaRestApi(this, "rest-test2", {
+      handler: hello,
+      deployOptions: {
+        accessLogDestination: new LogGroupLogDestination(restLogGroup2),
+      },
+    });
+
     const datadogCDK = new Datadog(this, "Datadog", {
       nodeLayerVersion: 39,
       pythonLayerVersion: 24,
-      forwarderArn: "<forwarder_ARN>",
+      forwarderArn: "arn:aws:lambda:sa-east-1:601427279990:function:datadog-forwarder-staging-org-2",
```

Comment:
We can probably just remove the actual forwarder arn and leave it with the place holder

Reply:
I know you never added an api key but FYI if you ever do I would set up an env variable for in your terminal so that in the future you dont accidentally commit an api key.